src/sql/parts/connection/connectionDialog/connectionDialogService.ts

```diff
@@ -336,6 +336,7 @@ export class ConnectionDialogService implements IConnectionDialogService {
 		// this solves the most common "hard error" that we've noticed
 		const helpLink = 'https://aka.ms/sqlopskerberos';
 		let actions: IAction[] = [];
+		let messageIsHtml = false;
```

Comment:
This seems like a lot of changes for 1 use case. Maybe we should look into changing this code to not need html messages to get around this.

Reply:
Good call. Turns out the only HTML use was for line breaks which we can still get by setting `innerText` so I just updated the dialog to always use that.